### ISSUE TITLE
Is there an option for port?

### ISSUE THREAD

**Issue by crates51** [KenN7/vim-arsync#12]
Hello,

I've tried adding  :24 to remote host
`remote_host     10.xxx.xx.xxx:24`
but is still said port 22:
`| ssh: connect to host 10.xxx.xx.xxx port 22: Connection refused^
`
How can i change the port to connect to with ssh?

Regards!

**Comment by KenN7** [comment]
Hi,

For now there is no option for the port.
It is a good suggestion and I may add it in the near future.

As a quick workaround i can propose to edit your `~/.ssh/config`
```
Host server1
     HostName 10.xxx.xx.xxx
     Port 24
```

**Comment by crates51** [comment]
thanks !

**Comment by crates51** [comment]
Thank you for the quick reply!

I managed to make it work by modifying(very ugly and hardcoding) vim-arsync.vim at line 71:
from:
`let l:cmd = [ 'rsync', '-avzhe', 'ssh', l:user_passwd . l:conf_dict['remote_host'] . ':' . l:conf_dict['remote_path'] . '/', l:conf_dict['local_path'] . '/']
`
to:
`let l:cmd = [ 'rsync', '-avzhe', 'ssh -p 24', l:user_passwd . l:conf_dict['remote_host'] . ':' . l:conf_dict['remote_path'] . '/', l:conf_dict['local_path'] . '/']
`

basically adding "-p 24" after ssh
